src/lib.rs

```diff
@@ -65,7 +65,6 @@ use std::hash::Hash;
 use std::fmt::Write;
 #[cfg(feature = "use_std")]
 type VecIntoIter<T> = ::std::vec::IntoIter<T>;
-#[cfg(feature = "use_std")]
 use std::iter::FromIterator;
```

Comment:
Is the `use_std` condition intentionally gone?

Reply:
No, I'm embarrassed, I corrected it.

Reply:
Never mind! :)